parser/mongodatastore.go

```diff
@@ -144,7 +144,7 @@ func (mongo *MongoDatastore) getCollectionMap(data *ImportedData) (*collectionMa
 
 	//iterate over indices to save RAM
 	//nolint: golint
-	for i, _ := range mongo.analyzedDBs {
+	for i := range mongo.analyzedDBs {
```

Comment:
I assume this is ok in Go, but does anyone else know for sure?  I'm used to Python where this would be a problem so it tripped my spidey-sense.